index.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     'mocha/no-pending-tests': 'error',
     'mocha/no-skipped-tests': 'warn',
     'mocha/valid-test-description': 'error',
+    'no-template-curly-in-string': 'off',
```

Comment:
The change to this rule in `ember-frost-bunsen` would be made here: https://github.com/ciena-frost/ember-frost-bunsen/blob/eb2c26475ba2245b4dd3f6e97cf914a81ca90d86/.eslintrc#L6